packages/taquito-core/src/interface/http_status_code.ts

```diff
@@ -0,0 +1,378 @@
+/**
+ * Hypertext Transfer Protocol (HTTP) response status codes.
+ * @see {@link https://en.wikipedia.org/wiki/List_of_HTTP_status_codes}
+ */
```

Comment:
wouldn't this make sense to keep at http utils instead?

Reply:
i don't think this should live in core, the only times we will check for http codes will be in side the http util package

Reply:
http_status_code is used multiple packages and it's used in HttpResponseError under @taquito/core.  Tried to make @taquito/core a standalone not importing from http utils is the reason that I moved it but understand if it's not ideal too.

Reply:
i agree it makes more sense in http-utils but see currently how this move is necessary to remove its dependency in multiple packages 
if our util packages do not depend on any of our other base level packages then it should stay in http-utils but until then it would make sense to keep it out of http-utils until those dependencies are sorted out